From 76091445db8b49a7d78504b47eb34fcbfbb89567 Mon Sep 17 00:00:00 2001
From: Elvis Pranskevichus <elvis@magic.io>
Date: Fri, 1 May 2020 20:02:39 -0700
Subject: [PATCH] Properly declare uuid_to_str and uuid_to_hex as static

Without `static` the toolchain might get confused and this might lead to
`undefined symbol: uuid_to_hex` errors.
---
 tohex.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/tohex.h b/tohex.h
index 3d2f5ad..60dd820 100644
--- a/tohex.h
+++ b/tohex.h
@@ -27,7 +27,7 @@
     }
 
 
-void inline
+static inline void
 uuid_to_str(const char *source, char *dest)
 {
     HEX_PRELUDE
@@ -44,7 +44,8 @@ uuid_to_str(const char *source, char *dest)
     HEX_2_BYTES(source + 14, dest + 32)
 }
 
-void inline
+
+static inline void
 uuid_to_hex(const char *source, char *dest)
 {
     HEX_PRELUDE
